src/views/drivingLicence/licence-issuer.html

```diff
@@ -45,12 +40,18 @@
                     statusCode: '200', // Access status code
                     englishPageTitle: '{{translate("licence-issuer.title")}}',
                     taxonomy_level1: 'CRI', // Access taxonomy level 1
-                    taxonomy_level2: 'licence-issuer', // Access taxonomy level 2
+                    taxonomy_level2: 'driving licence', // Access taxonomy level 2
                     content_id: '001',
                     logged_in_status: true,
                     dynamic: false
                 });
         });
     </script>
     {{ super ()}}
+{% endblock %}
+
+{% set footerNavItems = translate("govuk.footerNavItems") %}
```

Comment:
Correct me if i'm wrong but is it worth putting the variable names at the top with` {% set hmpoPageKey = "licence-issuer" %}`

Reply:
You're correct, but I simply followed the pattern they employed in their other Nunjucks file to refactor mine.